examples/em-server-multi-tier-cronjob/analysis/start_cron.sh

```diff
@@ -29,4 +29,4 @@ pip install devcron
 # launch the cronjob
 echo "Launch the cronjob"
 # while true; do sleep 30; done;
-devcron ../crontab >> /var/log/cron.console.stdinout 2>&1
+devcron ./crontab >> /var/log/cron.console.stdinout 2>&1
```

Comment:
I am pretty sure that this doesn't work

You are in the e-mission-server directory at this point, and the crontab is copied to `/usr/src/app`

```
COPY crontab /usr/src/app/crontab
```

Can you indicate how you tested this?

The docker-compose output has

```
em-server-multi-tier-cronjob-analysis-server-1  | Launch the cronjob
```

But nothing indicating that the cronjob was actually invoked.

You could check the logs to see if the intake scripts were actually launched every hour
I bet they are not.
Although I'm also curious about what failed and caused you to want to change this.

Reply:
So I think this is the same issue that I commented on in https://github.com/e-mission/e-mission-server/pull/875.

There is no longer a `e-mission-server` directory, so both the server source and crontab exist in `/usr/src/app`. When I'm in the correct dir to access the e-mission files, I'm also in the correct dir the access the `crontab` file.

The compose output having `em-server-multi-tier-cronjob-analysis-server-1  | Launch the cronjob` but no output about it after that is what I was seeing when I ran the original. I'll look at the logs.

It did take me a second to realize what was going wrong when the original line `devcron ../crontab >> /var/log/cron.console.stdinout 2>&1` ran and failed. It just said exited with code 1. Like I said in the other comment, I hadn't really realized that I had changed the directory structure. Perhaps this is a reason to keep it the way it was? Create some distinction between the server files themselves and these other scripts that help run the server.

Reply:
Yes would be good to keep the crontab out of the server code - in other words, keep `/usr/src/app` for the app only.

Reply:
Ok. So I will keep the server code directly in `/usr/src/app` and create some `/usr/src/run_server` dir for the supplemental scripts in `webbapp` and `analysis`.

Reply:
or you can even put them in `.docker`
Just don't want them in the main server directory.